cheat/cheatsheets/jq

```diff
@@ -0,0 +1,13 @@
+# Pretty print the json with color
+jq -C "." < filename.json
```

Comment:
I think that would be better, yes. You don’t really have to think about color in `jq`, so I think it would be best to hide that option from people, so they don’t start thinking about it unnecessarily.